backend/tests/apps/common/management/commands/clear_cache_test.py

```
@@ -0,0 +1,70 @@
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'test-cache',
+    }
+})
+def setup_cache():
+    cache.clear()
+    yield
+    cache.clear()
+
+
+@override_settings(CACHES={
+    'default': {
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)